tests/ert/unit_tests/forward_model_runner/test_fm_dispatch.py

```diff
@@ -405,3 +407,94 @@ def raises(*args, **kwargs):
 
     _report_all_messages(iter([message1, message2]), [reporter])
     reporter.report.assert_called_once_with(message1)
+
+
+@pytest.mark.integration_test
+async def test_fm_dispatch_sends_exited_event_with_terminated_msg_on_sigterm(
+    tmp_path, unused_tcp_port
+):
+    os.chdir(tmp_path)
+    with open("dummy_executable", "w", encoding="utf-8") as f:  # noqa: ASYNC230
+        f.write(
+            """#!/usr/bin/env python
+import time
+time.sleep(180)"""
+        )
+
+    executable = os.path.realpath("dummy_executable")
+    os.chmod("dummy_executable", stat.S_IRWXU | stat.S_IRWXO | stat.S_IRWXG)
+
+    fm_description = {
```

Comment:
I think it should be possible to remove all empty settings here (empty dicts or None). These are `@dataclass` objects now.